src/clj/rems/handler.clj

```diff
@@ -110,6 +110,15 @@
 (def ^:private resource-handler
   (route/resources "/" {:root "public"}))
 
+(defn- dev-js-handler
```

Comment:
Do we need the same for CSS or was it that Figwheel does the busting there?

Reply:
Hmm, it looks like CSS live-reloading isn't working for me currently. If I reload my browser it does get the fresh screen.css, but editing rems.css.styles and restarting the mount component doesn't do anything.

In any case, we need the `dev-js-handler` to avoid hitting the `(wrap-cacheable ... resource-handler)` which would serve app.js otherwise. CSS is always served by `styles/css-routes` which only serves uncacheable results.

... That should probably be fixed. I'll look into it.

Reply:
Fixed in follow-up PR